contracts/ShortsReelsVideos.sol

```diff
@@ -4,31 +4,52 @@ pragma solidity ^0.8.17;
 
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "@openzeppelin/openzeppelin-contracts/contracts/security/ReentrancyGuard.sol";
 
-import "./interfaces/IVanityURL.sol";
-
-contract ShortsReelsVideos is
+contract ContentManager is
     OwnableUpgradeable,
-    PausableUpgradeable,
-    ReentrancyGuardUpgradeable
+    PausableUpgradeable
+    // ReentrancyGuardUpgradeable
 {
     /// @dev VanityURL contract address
     IVanityURL public vanityURLAddress;
 
+    struct ContentRecord {
+        address owner;
```

Comment:
I think the owner is always the same as the D1DCV2 renter.
https://github.com/harmony-one/.1.country/blob/v1.1/contracts/contracts/D1DCV2.sol#L376